app/src/ui/app.tsx

```diff
@@ -348,6 +348,41 @@ export class App extends React.Component<IAppProps, IAppState> {
         return this.openCurrentRepositoryInExternalEditor()
       case 'select-all':
         return this.selectAll()
+      case 'show-release-notes-popup':
+        return this.props.dispatcher.showPopup({
```

Comment:
Is it elegant to have the notes for a fake release sitting directly in the `switch` statement here? Shall I move them out to another spot? If so, where is a good place?

Reply:
I'm not quite sure where the best place to put this is, but I'd love for this to be stripped out of the build when we generate the production bits. We use the `__DEV__` global for this so that webpack can then eliminate dead/unreachable code, but I'm not sure how best to do that within a `switch` statement.

Reply:
I can have the `case` block call a helper function (`showFakeReleaseNotesPopup`?), move all this into the implementation of said helper, and wrap the whole body of the helper with `if (__DEV__)`. Like so:
<img width="593" alt="fake" src="https://user-images.githubusercontent.com/3772093/50118671-4c514f80-0205-11e9-8f1a-7d0e1fe97eb3.png">

`showFakeReleaseNotesPopup` becomes an empty/no-op function in `prod` builds. Its declaration is still in the package, and it still doesn't convey to a human that it's dev-only functionality until you actually look at its implementation, which isn't ideal. But if the concern is around having the fake release notes in the build, this does accomplish leaving them out.

Reply:
Yeah, let's go with this approach. For production builds this should be a no-op, as the menu shouldn't be accessible to users, but for development mode this can work as currently designed.